pkg/monitor-hw/cmd/root.go

```diff
@@ -0,0 +1,91 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+
+	"github.com/cybozu-go/log"
+	"github.com/cybozu-go/setup-hw/config"
+	"github.com/cybozu-go/setup-hw/lib"
+	"github.com/cybozu-go/well"
+	"github.com/spf13/cobra"
+)
+
+var opts struct {
+	listenAddress string
+	redfishRoot   string
+	interval      int
+	resetInterval int
+}
+
+const (
+	defaultPort          = ":9105"
+	defaultRedfishRoot   = "/redfish/v1"
+	defaultInterval      = 60
+	defaultResetInterval = 3600
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "monitor-hw",
+	Short: "monitor-hw daemon",
+	Long:  `monitor-hw is a daemon to monitor server statuses.`,
+
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
When you use `RunE`, you should almost always set `cmd.SilenceUsage` to `false`.
Otherwise, any error will be displayed with usage strings.
https://godoc.org/github.com/spf13/cobra#Command

Please confirm it by yourself.